encoders/ffmpeg_test.go

```diff
@@ -140,7 +140,7 @@ var _ = Describe("FFmpeg Encoder", func() {
 					},
 				},
 				Status:           types.JobCreated,
-				Details:          "0%",
```

Comment:
I agree with you that this will be fixed on #152, but I think we can use the case for this PR now. When we solve the audio issue, we'll need to update the test anyway.